src/views/Privacy.vue

```diff
@@ -6,171 +6,89 @@
     <div class="container wrapper mt-2">
       <div class="row content">
         <div class="col-sm-12">
-          <h1>PRIVACY POLICY</h1>
+          <h1>{{ $t('views.Privacy.title') }}</h1>
 
           <p>
-            The present Privacy Policy was created on 13th February 2020.
+            {{ $t('views.Privacy.subtitle') }}
           </p>
 
-          <p>
-            I. Introduction
-            Superhero.com LVC, Dr. Grass Str 12, 9490 Vaduz,
-            Liechtenstein (“Superhero.com”, “we”, “us” or “our”)
-            is a company which takes your privacy and the security
-            of your information as a User very seriously.
-          </p>
+          <div class="first">
+            <h5><b>{{ $t('views.Privacy.first') }}</b></h5>
```

Comment:
IMHO this should accept the whole block as HTML (of course json escaped) and put as raw html